chore(): task #1005

SNKz0610 opened the pull request:
Adding Android homework 10/05/2021 about first month ending by using git flow.Closes task #1005

SNKz0610 committed:
chore(): task #1005
Adding Android homework 10/05/2021 about first month ending by using git flow.Closes task #1005

linhvnguyen9 commented:
#1 Phần tạo nhánh theo Git flow b làm khá ok r, bao gồm cách tách nhánh (master, develop, feature,...), cách tạo pull request. Còn 1 góp ý nhỏ chỗ đặt tên nhánh `feature/feature_register`. Ở đây b đã ghi feature ở đằng trước r, vì thế b ko cần phải ghi feature ở phần sau dấu gạch nữa

linhvnguyen9 commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -0,0 +1,97 @@
+        android:layout_marginTop="10dp"
+        android:layout_width="60dp"
+        android:layout_height="40dp"
+        android:backgroundTint="#ffffff"
```
1 tip nhỏ cho b ở đây, nếu b muốn cái nút đó trong suốt (ko có viền với nền ở quanh), b có thể xài `android:background="@android:color/transparent"`

linhvnguyen9 commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -0,0 +1,97 @@
+        android:src="@drawable/continuewithgmail"
+        />
+
+    <ImageButton
```
Ở đây b đang để phần bên trong của nút (icon và ảnh) dưới dạng 1 ảnh duy nhất. Điều gì xảy ra nếu app của bạn hỗ trợ 20 ngôn ngữ? Bạn sẽ phải sửa như thế nào để dịch hết dòng chữ chú thích cho từng nút kia?

linhvnguyen9 commented on app/src/main/java/com/example/register/MainActivity.java:
```diff
@@ -0,0 +1,14 @@
+
+import android.os.Bundle;
+
+public class MainActivity extends AppCompatActivity {
```
Project của b đang code = Java, có thể b đang để mặc định ngôn ngữ là Java nên quên mất :)))))). Các part đằng sau của mình yêu cầu code = Kotlin, b thử tìm cách đổi, ít nhất là file MainActivity này sang Kotlin nhé!

linhvnguyen9 reviewed (changes requested):
B check review, sửa code và push lên nhánh feature của b nhé!

SNKz0610 committed:
chore(): task#1105
Convert class MainActivity from Java to Kotlin.
Fixing buttons, incluing change ImageButto to Button at 4 functions of continue with.... and change the background into transparency
Closes #task1105

SNKz0610 commented:
Fixing something belong to the comment
